src/OpenTelemetry.Instrumentation.SqlClient/CompatibilitySuppressions.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
@rajkumar-rangaraj, @alanwest, what is your opinion here?
Can we ignore this? .NET Standard target has broader API than .NET Framework.

Reply:
I'm fine with removing it. @alanwest Your thoughts?

Reply:
> Methods removed int his PR are noop in .net fx

Then yes it sounds like it's safe to keep the APIs in the netstandard build.